storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/Query.java

```diff
@@ -103,9 +102,7 @@ public String toString() {
     /**
      * Builder for {@link Query}
      */
-    public static class Builder {
-
-        Uri uri;
+    public static final class Builder {
```

Comment:
But how he would apply his custom class?

вс, 26 апр. 2015, 14:54, Dmitrii Nikitin <notifications@github.com>:

> In
> storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/Query.java
> <https://github.com/pushtorefresh/storio/pull/302#discussion_r29109066>:
>
> > @@ -103,9 +102,7 @@ public String toString() {
> >      /**
> >       * Builder for {@link Query}
> >       */
> > -    public static class Builder {
> > -
> > -        Uri uri;
> > +    public static final class Builder {
>
> Why did you decide to make it final? Can it be useful for user to extend
> it for some custom behavior. For example receive String instead of Uri?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/302/files#r29109066>.
>